rca/utils/models.py

```diff
@@ -111,6 +111,48 @@ class Meta:
     panels = [PageChooserPanel("page")]
 
 
+class RelatedStaffPageWithManualOptions(Orderable):
```

Comment:
Could `source_page` from the concrete versions of this be added to this model instead and be linked to `Page` (and this model made not abstract), so you don't need to keep creating page-specific concrete models?

Reply:
hmmm, maybe my understanding of this isn't spot up. I've done it this way so a project can have multiple related staff but also this is used across other page types, so making the base abstract has help me avoid carrying around the migration. Might be worth discussing this though